README.md

```diff
@@ -0,0 +1,65 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project 
+
+# Community Development 
+
+![screenshot](img/ScreenCap.png)
+
+>This website if a community development website dedicated for my local city
+
+>
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+
+## Live presentation
+
+- https://www.loom.com/share/8feee13007f2414182ba4ef53aee9173
+
+### Usage
+
+ - to get a local copy up and running follow these simple example steps.
+
+- Open terminal
+
+- Clone this project by the command `git clone https://github.com/mutinhiri/CapstonePr1`
+
+- `Cd feature` folder
+
+- Open `index.html` in your local browser or using Live Server in Visual Studio Code.
+
+# Live Demo
+
+[Community development] (https://mutinhiri.github.io/capstonepr)
+
+
+## Authors
+
+👤 **Bunbee**
+
+- GitHub: [@mutinhiri](https://github.com/mutinhiri)
+- Twitter: [@Bunbee2271](https://twitter.com/@Bunbee2271)
+- LinkedIn: [barnabas mutinhiri](https://linkedin.com/in/bunbee)
+
+# Credits : Cindy Shin
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
**Your README file is missing this important point:**
- "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin the author of the original design, as required by the Creative Commons License.